### PR TITLE
Feature/add citations to ao canonical page

### PR DESCRIPTION
**Pull request by anthonygarvan**

### CODE REVIEW

File: openfecwebapp/templates/legal-advisory-opinion.html [hunk 1]
```diff
@@ -62,6 +54,37 @@ <h2 class="t-ruled--bold t-ruled--bottom">Documents</h2>
       <h2 class="t-ruled--bold t-ruled--bottom">Entities</h2>
       Coming soon.
     </section>
+    <section class="content__section">
+      <aside id="legal-citations" class="sidebar sidebar--primary">
+          <h4 class="sidebar__title">Legal citations</h4>
+          <div class="sidebar__content">
+            <div class="grid grid--2-wide">
+                <div class="grid__item">
+                  <p class="t-bold">This opinion cites earlier opinions</p>
```
Comment (vrajmohan) [inline openfecwebapp/templates/legal-advisory-opinion.html]: Shouldn't this be inside the `if` 2 lines below?

File: openfecwebapp/templates/legal-advisory-opinion.html [hunk 2]
```diff
@@ -62,6 +54,37 @@ <h2 class="t-ruled--bold t-ruled--bottom">Documents</h2>
...
+                    {% for citation in advisory_opinion.citations %}
+                      <p><a href="/legal/advisory-opinions/{{ citation.no }}">AO {{ citation.no }}</a><br><i>{{ citation.name }}</i></p>
+                    {% endfor %}
+                    {% else %}
+                      <p>Does not cite other opinions</p>
+                    {% endif %}
+                  </div>
+                </div>
+                <div class="grid__item">
+                  <p class="t-bold">This opinion is cited by these later opinions</p>
```
Comment (vrajmohan) [inline openfecwebapp/templates/legal-advisory-opinion.html]: Shouldn't this also be inside the `if` 2 lines below?

Comment (anthonygarvan) [inline openfecwebapp/templates/legal-advisory-opinion.html]: No, that is the title of the section, not an error message: screen shot 2017-01-24 at 3 05 31 pm

Comment (anthonygarvan) [inline openfecwebapp/templates/legal-advisory-opinion.html]: Oh I see what you're asking, let me pull in @emileighoutlaw

**Pull request merged**